adapters/AppLovin/AppLovinAdapter/RTB/GADMAppLovinRtbRewardedRenderer.m

```diff
@@ -0,0 +1,149 @@
+//
+//  GADMAppLovinRtbRewardedRenderer.m
+//  Adapter
+//
+//  Created by Christopher Cong on 7/17/18.
+//  Copyright © 2018 Google. All rights reserved.
+//
+
+#import "GADMAppLovinRtbRewardedRenderer.h"
+#import "GADMAdapterAppLovinUtils.h"
+#import "GADMAdapterAppLovinConstant.h"
+#import "GADMAdapterAppLovinExtras.h"
+
+#import <AppLovinSDK/AppLovinSDK.h>
+
+@interface GADMAppLovinRtbRewardedRenderer () <GADMediationRewardedAd, ALAdLoadDelegate, ALAdDisplayDelegate, ALAdVideoPlaybackDelegate, ALAdRewardDelegate>
+
+/// Data used to render an RTB rewarded ad.
+@property (nonatomic, strong) GADMediationRewardedAdConfiguration *adConfiguration;
+
+/// Callback object to notify the Google Mobile Ads SDK if ad rendering succeeded or failed.
+@property (nonatomic, copy) GADRewardedRenderCompletionHandler renderCompletionHandler;
+
+/// Delegate to notify the Google Mobile Ads SDK of rewarded presentation events.
+@property (nonatomic, strong, nullable) id<GADMediationRewardedAdEventDelegate> adEventDelegate;
+
+/// Controlled Properties
+@property (nonatomic, strong) ALSdk *sdk;
+@property (nonatomic, strong) ALIncentivizedInterstitialAd *incentivizedAd;
+@property (nonatomic, strong) ALAd *ad;
+@property (nonatomic, assign) BOOL fullyWatched;
+@property (nonatomic, strong) GADAdReward *reward;
+
+@end
+
+@implementation GADMAppLovinRtbRewardedRenderer
+
+- (instancetype)initWithAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
+                      completionHandler:(nonnull GADRewardedRenderCompletionHandler)handler {
```

Comment:
What's the point of specifying `nonnull` when we are already assuming it from interface?